CMakeLists.txt

```diff
@@ -418,6 +417,7 @@ add_executable(main_ui
 set_property(DIRECTORY APPEND PROPERTY ADDITIONAL_MAKE_CLEAN_FILES ${GENERATED_MOC_FILES})
 
 qt5_use_modules(main_ui Core Gui Widgets)
+add_dependencies(main_ui openssl)
```

Comment:
Yes I needed to add it somewhere to make sure it will compile. I'll move it to `veles_client` since it will be the place which uses ssl.